### ISSUE TITLE
bug in v0.3.0 hauler version doesn't correctly specify version of hauler

### ISSUE THREAD

**Issue by neoakris** [hauler-dev/hauler#128]
**Environmental Info:**  
linux/amd64

**Hauler Version:**
0.3.0

**System CPU architecture, OS, and Version:**
OS = Centos 7   
`uname -a`  
```console
Linux vm1 3.10.0-1160.59.1.el7.x86_64 #1 SMP Wed Feb 23 16:47:03 
UTC 2022 x86_64 x86_64 x86_64 GNU/Linux  
```

**Describe the bug:**
hauler version doesn't correctly specify the version of hauler

**Steps To Reproduce:**
On a x86_x64 based linux machine copy paste the following 4 commands:
```shell
curl -L https://github.com/rancherfederal/hauler/releases/download/v0.3.0/hauler_0.3.0_linux_amd64.tar.gz > hauler.tar.gz
tar -xzf hauler.tar.gz 
sudo mv hauler /usr/bin/hauler && rm README.md
hauler version
```

**Expected behavior:**
expect to see version 0.3.0 specified in the output

**Actual behavior:**
`hauler version`
```console
GitVersion:  devel
GitCommit:   unknown
BuildDate:   unknown
GoVersion:   go1.17.6
Compiler:    gc
Platform:    linux/amd64
```

**Comment by zackbradys** [comment]
Bumping this issue. `hauler version` still does not show all information and does not show the Hauler Version. Additionally, there is no command alias such as `hauler -v`.

```yaml
zackbradys@Zacks-MacBook-Pro bin % hauler version
GitVersion:  devel
GitCommit:   unknown
BuildDate:   unknown
GoVersion:   go1.20.5
Compiler:    gc
Platform:    darwin/arm64
```

**Comment by zackbradys** [comment]
fixed in PR #136!
